sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusAsyncConsumer.java

```diff
@@ -29,14 +33,19 @@
     private final Flux<ServiceBusReceivedMessage> processor;
 
     ServiceBusAsyncConsumer(String linkName, ServiceBusReceiveLinkProcessor linkProcessor,
-        MessageSerializer messageSerializer, int prefetch) {
+        MessageSerializer messageSerializer, int prefetch, boolean autoLockRenewal,
+        Duration maxAutoLockRenewDuration, LockContainer<LockRenewalOperation> messageLockContainer,
+        Function<String, Mono<OffsetDateTime>> onRenewLock) {
         this.linkName = linkName;
         this.linkProcessor = linkProcessor;
         this.messageSerializer = messageSerializer;
-        this.processor = linkProcessor
+
+        this.processor = new ServiceBusRenewOperator(linkProcessor
```

Comment:
Is this operator required even if the `autoLockRenewal` is set to false? Also, this member variable should now be renamed to `renewOperator`.